### DISCUSSION TITLE
How to make 2 Grid.Rows with different ScaleTransform the same width?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#12017] [Q&A]
This is not necessarily a UNO question, but I figured I might ask anyway...

I'm pulling my hair since I can't figure out how to make 2 rows of a grid with different `ScaleTransform `the same width. Here is [a code snippet](http://playground.platform.uno/#dbdeda03)

And here is the problem as screenshot
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/233157607-ef284858-ef2f-404f-82b0-86e5c9e6dace.png">

I would appreciate any hint on how to resolve this nasty issue. Thanks in advance

**Comment by FrozDark** [comment]
~~Maybe you should try Viewbox?~~

You question doesn't make sense at all

**Comment by DierkDroth** [comment]
@FrozDark apologies if the issue is not clear. If your looking at the screenshot you will see:
- there are 2 rows with different scaling
- the row content of both rows (see source) should be 'long enough' to 'fill the complete width'
- however, it doesn't -> problem

**Reply by FrozDark** [reply]
> @FrozDark apologies if the issue is not clear. If your looking at the screenshot you will see:
> 
> * there are 2 rows with different scaling
> * the row content of both rows (see source) should be 'long enough' to 'fill the complete width'
> * however, it doesn't -> problem

Transform is not something that is included in a controls size measurements
It's not clear the approach you are trying to achieve
Instead of ScaleTransform, I would rather recommend using Viewbox. Or don't change ScaleX, ScaleY (or reset to 1) when appropriate

**Reply by DierkDroth** [reply]
Thanks @FrozDark. I did some reading on ViewBox, but fail to understand how this could help in my scenario above.

Would you mind applying the relevant changes to the code which you access by the link I provided above? Thanks in advance.
